recipes/pdfmm/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+CXX_FLAGS="${CXX_FLAGS} -fmessage-length=0 -fstack-protector-all -Ofast -D_FORTIFY_SOURCE=2"
+CXX_FLAGS="${CXX_FLAGS} -funwind-tables -fpic -fasynchronous-unwind-tables"
+
+# configure cmake
+rm -rf build
+cmake -B build \
+  -DCMAKE_PREFIX_PATH=${PREFIX} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_CXX_STANDARD=17 \
+  -DPDFMM_BUILD_STATIC=OFF \
+  -DPDFMM_USE_VISIBILITY=ON \
+  -DPDFMM_BUILD_LIB_ONLY=ON
+
```

Comment:
```suggestion
cmake -B build \
  -DCMAKE_PREFIX_PATH=${PREFIX} \
  -DCMAKE_BUILD_TYPE=Release \
  -DCMAKE_CXX_STANDARD=17 \
  -DPDFMM_BUILD_STATIC=OFF \
  -DPDFMM_USE_VISIBILITY=ON \
  -DPDFMM_BUILD_LIB_ONLY=ON \
  ${CMAKE_ARGS}

```
We have an environment variable that automatically sets conda-forge specific args for cmake.


https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cmake_args#using-cmake

Reply:
I didn't know about that! thank you for the explanation!